src/styles/utils/_mixins.scss

```
@@ -0,0 +1,5 @@
+@mixin square($block_side) {
+  width: $block_side;
+  height: $block_side;
+  border: 1px solid black;
+}
```

I would say it is clear enough this way
```suggestion
@mixin square($size) {
  width: $size;
  height: $size;
  border: 1px solid black;
}
```

---

edited to $size